test/core/transport/status_conversion_test.c

```diff
@@ -129,6 +132,7 @@ int main(int argc, char **argv) {
                              GRPC_STATUS_INTERNAL);
   HTTP2_ERROR_TO_GRPC_STATUS(GRPC_HTTP2_REFUSED_STREAM, after_deadline,
                              GRPC_STATUS_UNAVAILABLE);
+  sleep(1);
```

Comment:
can you add a comment explaining why a sleep here isn't a hack? In other words, why it is needed.